lisa/transformers/kernel_source_installer.py

```diff
@@ -27,7 +26,12 @@ class BaseModifierSchema(schema.TypedSchema, schema.ExtendableSchemaMixin):
 @dataclass_json()
 @dataclass
 class BaseLocationSchema(schema.TypedSchema, schema.ExtendableSchemaMixin):
-    ...
+    kernel_config_file: str = field(
```

Comment:
Also, this field should probably be in SourceInstallerSchema since it's purpose is not for specifying the location of the kernel source code.

Reply:
@squirrelsc what do you think of this?

Reply:
The BaseLocationSchema is for location of source code. It has local (existing code), repo (download source code from remote repo). Both of them need to support kernel configuration. It's not about where the source code is from.

Reply:
I moved kernel config file to source installer schema. It can be absolute path of node.

Reply:
Pushed it to source installer schema. Implemented the Child class of it for dom0 specific actions.